components/list/list-item-checkbox-mixin.js

```diff
@@ -12,8 +12,10 @@ export const ListItemCheckboxMixin = superclass => class extends SkeletonMixin(s
 			/**
 			 * **Selection:** Disables the input
 			 * @type {boolean}
+			 * @ignore
 			 */
-			disabled: { type: Boolean },
+			disabled: { type: Boolean }, // deprecated
+			selectionDisabled: { type: Boolean, attribute: 'selection-disabled' },
```

Comment:
The Daylight documentation pulls the component properties, slots and events directly from the component itself. That's why we have marked up the documentation for the other properties and why the `@ignore` was necessary on the deprecated component so it is no longer shown on the docs. We should add similar markup for this new property.